pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+requires = ["setuptools",
+            "wheel",
+            "cython>=0.29.21",
+            "numpy==1.16.0; python_version <= '3.7'",
+            "numpy==1.17.3; python_version == '3.8'",
+            "numpy==1.19.3; python_version == '3.9'",
+            "numpy==1.21.4; python_version =='3.10'",
+            "numpy; python_version >= '3.11'",
```

Comment:
Not sure if this breaks anything but there appears to be an extra comma at the end of the line.